src/qililab/platform/platform.py

```diff
@@ -768,7 +767,15 @@ def execute(
             if isinstance(instrument, QbloxModule):
                 instrument.desync_sequencers()
 
-        # FIXME: set multiple readout buses
+        # Flatten results if more than one readout bus was used for a qblox module
+        if len(results) > 1:
+            return QbloxResult(
+                integration_lengths=[length for result in results for length in result.integration_lengths],  # type: ignore [attr-defined]
+                qblox_raw_results=[raw_result for result in results for raw_result in result.qblox_raw_results],  # type: ignore [attr-defined]
+            )
+        if not results:
+            raise ValueError("There are no readout buses in the platform.")
+
```

Comment:
Why do we change this?

Reply:
same as above https://github.com/qilimanjaro-tech/qililab/pull/656 I'll add a changelog entry for this as well